File: App.Models/MeetingParticipant.cs
using App.Core.Models;

namespace App.Models
{
    public class MeetingParticipant : IAuditableEntity
    {
        public int MeetingId { get; set; }
        public int UserId { get; set; }
        public MeetingParticipantRole Role { get; set; }
        public bool Attended { get; set; } = false;
        public DateTime CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }

        // Navigation properties
        public virtual Meeting Meeting { get; set; } = null!;
        public virtual User User { get; set; } = null!;
    }
}

File: App.client/projects/proctor-app/scripts/models/meeting-participant.ts
import { User, MeetingParticipantRole } from './user';
import { Meeting } from './meeting';

export class MeetingParticipant {
  meetingId: number;
  userId: number;
  role: string;
  attended: boolean;
  createdOn: string;
  modifiedOn: string;
  meeting?: Meeting | null;
  user?: User | null;

  constructor() {
    this.meetingId = this.userId = 0;
    this.role = MeetingParticipantRole.Other;
    this.attended = false;
    this.createdOn = this.modifiedOn = '';
  }
}

